Add a new orientation of "inline-column"

jlubean opened the pull request:
Add a new orientation of "inline-column" that is similar to "inline" but has a column/stacked layout. Fix small icon to be centered vertically with the name.

The immediate need is on the search results of the new v8 Tree > Find page:

![image](https://user-images.githubusercontent.com/1156467/50358693-76fa1b80-0517-11e9-9a66-9ab1316dd474.png)

Here's a sample usage:

```<fs-tree-person icon-size="small" person-obj="[[personObj]]"  orientation="inline-column" no-bold-name no-lifespan open-person-card></fs-tree-person>```

timothydm42 commented on fs-person.html:
```diff
@@ -168,15 +168,22 @@
     }
 
-    fs-person[orientation="inline"] .fs-person__vitals {
+    fs-person[orientation="inline"] .fs-person__vitals, {
```
omit comma here please

timothydm42 commented:
and example please in demo

jlubean commented:
I added it to the demo and removed the comma from the CSS.  I also fixed the small icon to be vertically aligned with the name.

![image](https://user-images.githubusercontent.com/1156467/50377267-48dc1080-05d7-11e9-854e-9a88dc1ab914.png)
